Completed prep-javascript-conditionals exercise

kelvin-lao-01 opened the pull request:
![Screen Shot 2022-08-10 at 2 33 35 PM](https://user-images.githubusercontent.com/48492707/184024376-ea9dfaec-9116-447f-86d7-01fc07444ebe.png)

thebearingedge commented on prep-javascript-conditionals/index.html:
```diff
@@ -0,0 +1,87 @@
+console.log(student1Score);
+
+var student2Score = '100';
+if (student2Score < 60) {
+  student2Score = 'F';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 70) {
+  student2Score = 'D';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 80) {
+  student2Score = 'C';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 90) {
+  student2Score = 'B';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 100) {
+  student2Score = 'A';
+} else {
+  student2Score = 'A++';
+}
```
Hi Kelvin, you'll need some `else if` blocks in here so that scores like 70 or 80 don't get turned into A++. 

```js
if (student2Score < 60) {

} else if (student2Score < 70) {

} else if (student2Score < 80) {

// etc etc
```

kelvin-lao-01 replied:
![Screen Shot 2022-08-14 at 10 06 17 AM](https://user-images.githubusercontent.com/48492707/184547490-8b9ffe55-8703-410f-88e9-476bdf9ea6a5.png)

thebearingedge commented on prep-javascript-conditionals/index.html:
```diff
@@ -0,0 +1,96 @@
+var student2Score = '100';
+if (student2Score < 60) {
+  student2Score = 'F';
+  } else if (student2Score < 100) {
+  student2Score = 'F';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 70) {
+  student2Score = 'D';
+} else if (student2Score < 70) {
+  student2Score = 'D';
+} else {
+  student2Score = 'A++';
+
+if (student2Score < 80) {
+  student2Score = 'C';
+  } else if (student2Score < 80) {
+  student2Score = 'C';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 90) {
+  student2Score = 'B';
+  } else if (student2Score < 90) {
+  student2Score = 'B';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 100) {
+  student2Score = 'A';
+  } else if (student2Score < 100) {
+  student2Score = 'A';
+} else {
+  student2Score = 'A++';
+}
+console.log(student2Score);
```
Hi Kelvin, try combining these into one `if ... else if ... else` "ladder"

https://web1.eng.famu.fsu.edu/~haik/met.dir/hcpp.dir/notes.dir/cppnotes/node41.html#:~:text=A%20common%20programming%20construct%20that,of%20the%20ladder%20is%20bypassed.

thebearingedge commented on prep-javascript-conditionals/index.html:
```diff
@@ -0,0 +1,96 @@
+}
+console.log(student2Score);
+
+var season = 'summer';
+if (season === 'summer') {
+  season = 'its hot today';
+}
+
+if (season === 'spring') {
+  season = 'the flowers are blooming';
+}
+
+if (season === 'autumn') {
+  season = 'the leaves are changing colors';
+}
+
+if (season === 'winter') {
+  season = 'its cold today';
+} else {
+  season = 'please enter a valid season';
+}
+console.log(season);
```
This part too, try combining it into one `if ... else if ... else` ladder.

thebearingedge commented on prep-javascript-conditionals/index.html:
```diff
@@ -0,0 +1,106 @@
+if (student2Score < 60) {
+  student2Score = 'F';
+} else if (student2Score < 70) {
+  student2Score = 'F';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 70) {
+  student2Score = 'D';
+} else if (student2Score < 80) {
+  student2Score = 'D';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 80) {
+  student2Score = 'C';
+} else if (student2Score < 90) {
+  student2Score = 'C';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 90) {
+  student2Score = 'B';
+} else if (student2Score < 100) {
+  student2Score = 'B';
+} else {
+  student2Score = 'A++';
+}
+
+if (student2Score < 100) {
+  student2Score = 'A';
+} else if (student2Score < 100) {
+  student2Score = 'A';
+} else {
+  student2Score = 'A++';
+}
+console.log(student2Score);
```
Hi Kelvin, try combining all of these in to one set of `if ... else if ... else`

```js
// partial example

if (student2Score < 60) {
  console.log('F');
} else if (student2Score < 70) {
  console.log('D');
} ... {
} else {
  console.log('A++');
}

```

You should have `else if` statements for each level of grade between `'F'` and `'A++'`.

thebearingedge commented on prep-javascript-conditionals/index.html:
```diff
@@ -0,0 +1,106 @@
+}
+console.log(student2Score);
+
+var season = 'summer';
+if (season === 'summer') {
+  season = 'its hot today';
+}
+
+if (season === 'spring') {
+  season = 'the flowers are blooming';
+}
+
+if (season === 'autumn') {
+  season = 'the leaves are changing colors';
+}
+
+if (season === 'winter') {
+  season = 'its cold today';
+} else {
+  season = 'please enter a valid season';
+}
+console.log(season);
```
Like mentioned above for the student score, try combining these all.

kelvin-lao-01 replied:
just saw this, got it

kelvin-lao-01 commented:
<img width="569" alt="Screen Shot 2022-08-15 at 10 18 41 PM" src="https://user-images.githubusercontent.com/48492707/184804105-a8121c29-d6aa-47c0-83ea-7fcbf0c11351.png">
<img width="548" alt="Screen Shot 2022-08-15 at 10 18 57 PM" src="https://user-images.githubusercontent.com/48492707/184804109-bfab74ec-24d3-4cba-9fbb-949cc44ccabe.png">

Okay I have updated the studen2score with else if / else + logging. I have also updated the seasons with else & if + logging. Since its just referring to seasons I wasn't sure if it needed an else if since its pretty straight forward based on what the assignment was asking in regards to the seasons matching. Please let me know if there's anything else I need to change.

thebearingedge reviewed (changes requested):
continued conversation in Slack regarding combining all of the if / else if / else checks for the different possible grades
